### DISCUSSION TITLE
lower resolution images are garbled, is this a stable diffusion flaw or can i fix it?

### DISCUSSION THREAD

**Discussion by TalonZane** [carson-katri/dream-textures#165] [Q&A]
my desired texture size is ~64x64 but the results i get are not really tangible unless i go up to 512x512, which takes a while on my computer due to a wacky gpu

**Comment by TiagoTiago** [comment]
The AI was not trained on lower resolutions, so it doesn't quite know how to get good results with too few pixels.
